AUTHORS.md

```diff
@@ -67,5 +68,8 @@ Authors are sorted alphabetically.
 * XuehaiPan
 * astaric
 * bkreider
+* conda-bot
+* dependabot[bot]
```

Comment:
```suggestion
```
We should remove the bots, correct?

Reply:
They'll keep popping up so tbh I don't really mind them here, I only remove them from the changelog, and not always.

The cool thing would be to have an `exclude` option in `rever.xsh` but I didn't look into it.